docs/source/models.rst

```diff
@@ -213,85 +213,14 @@ at hand, such as "What inputs does it expect?" and "What output does it produce?
 include the following additional metadata about model inputs, outputs and params that can be used by
 downstream tooling:
 
-* :ref:`Model Inference Params <inference-params>` - description of params used for model inference.
-* :ref:`Model Signature <model-signature>` - description of a model's inputs, outputs and parameters.
+* :ref:`Model Signature <model-signature>` - description of a model's inputs, outputs and additional parameters for inference.
 * :ref:`Model Input Example <input-example>` - example of a valid model input.
 
-.. _inference-params:
-
-Model Inference Params
```

Comment:
Note from offline converstaion:

`Inference parameter` might not be a concept limited to signature, as we can directly pass it to `predict()` function. Instead of locating it under signature section as a sub section, we can have a top-level section for it, so that the signature section can just describe basic usage and linke to it for details.

Reply:
@serena-ruan  Probably not gonna address this in this PR, as it's hard to find alternative place to put inference params section in, with current doc strucutre. Ideally, we should have some introductory section that shows how to run inference with MLflow model, and describe about inference params concept there. 

Re-structuring Models page is must-do anyway, it's by-far too long. For now I prioritize Tracking page as I already started on similar restructuring for it. Then I can come back once we find the best shape there, and find better place for inference params.

Reply:
@BenWilson2 What is our plan with spliting models.rst file? And new folder structures settled down?

Reply:
Note from stand-up: We are trying to port contents out from `models.rst` into children pages and make `models.rst` just a hub and overview page. I'll raise another PR to split model signature (+probably a few more sections like schema enforcement) and put inference param section in appropriate location on the new pages.